src/main/java/servlets/Helper.java

```diff
@@ -0,0 +1,97 @@
+package servlets;
+import java.util.Stack;
+import java.util.StringTokenizer;
+
+public class Helper {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Helper' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4886196)